cmd/get/nodepools/flag.go

```diff
@@ -0,0 +1,36 @@
+package nodepools
+
+import (
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+const (
+	flagAllNamespaces = "all-namespaces"
+	flagClusterID     = "cluster-id"
+)
+
+type flag struct {
+	AllNamespaces bool
+	ClusterID     string
+
+	config genericclioptions.RESTClientGetter
+	print  *genericclioptions.PrintFlags
+}
+
+func (f *flag) Init(cmd *cobra.Command) {
+	cmd.Flags().BoolVarP(&f.AllNamespaces, flagAllNamespaces, "A", false, "If present, list the requested object(s) across all namespaces. Namespace in current context is ignored even if specified with --namespace.")
```

Comment:
is no already defined the kubectl-specific ones?

Reply:
Unfortunately no... we do the same for the `get clusters` command, as well
https://github.com/giantswarm/kubectl-gs/blob/bc1138ebb28cd0e1125329961f240520c5dd387b/cmd/get/clusters/flag.go#L20